src/viewers/PictureInPictureViewer.js

```diff
@@ -22,7 +22,7 @@ import {
  * @param {Boolean} props.overviewOn Whether or not to show the OverviewView.
  * @param {Object} props.hoverHooks Object including the allowable hooks - right now only accepting a function with key handleValue like { handleValue: (valueArray) => {} } where valueArray
  * has the pixel values for the image under the hover location.
- * @param {number} props.initialViewState Object like { target: [x, y, 0], zoom: -zoom } for initializing where the viewer looks (optional - this is inferred from height/width/loader
+ * @param {Array} [props.viewStates] Array of objects like [{ target: [x, y, 0], zoom: -zoom, id: 'detail' }] for setting where the viewer looks (optional - this is inferred from height/width/loader
```

Comment:
Let's export DETAIL_VIEW_ID and OVERVIEW_VIEW_ID and update this docs section to be `[{ target: [x, y, 0], zoom: -zoom, id: DETAIL_VIEW_ID }]`

Reply:
Done.